docs/usage.md

```diff
@@ -13,7 +13,7 @@
 The typical command for running the pipeline is as follows:
 
 ```bash
-nextflow run nf-core/pangenome --input '*_R{1,2}.fastq.gz' -profile docker
+nextflow run nf-core/pangenome --input 'data/*.fa.gz' -profile docker
```

Comment:
I would not use the data folder here.
No users should be requested to have a data folder.